src/components/notes/EditorCorner.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Icon from 'cozy-ui/react/Icon'
+import Tooltip from 'cozy-ui/react/Tooltip'
+
+import SharingWidget from 'components/notes/sharing'
+
+const EditorCorner = ({ doc, isPublic, isReadOnly }) => {
+  if (!isPublic) return <SharingWidget file={doc.file} />
+  else if (isReadOnly)
+    return (
+      <Tooltip title={'This note is in read-only mode.'}>
+        <Icon icon="lock" color="var(--coolGrey)" />
```

Comment:
Nit: It would good not to have to hard code the icon color here. This hardcoded color would not work for example if we code a dark mode. Maybe we could achieve this with a prop `textIcon` that would set its color to `var(--iconTextColor)` that would be set by default to `var(--coolGrey)` ?

See https://github.com/cozy/cozy-ui/issues/1475 also.

Reply:
Good point, I ended up using `primaryTextColor` because the icon has no text next to it by default, but `iconTextColor ` is a good idea re. the cozy-ui issue.